src/gomod_parser.rs

```diff
@@ -0,0 +1,14 @@
+use regex::Regex;
+use std::collections::HashMap;
+
+//parses go.mod file fetching github/gitlab unique link
+pub fn parse(file: String) -> Vec<String> {
```

Comment:
сделай им скоуп видимости `crate`, плиз

```suggestion
pub(crate) fn parse(file: String) -> Vec<String> {
```